src/ontology/mondo-edit.obo

```diff
@@ -22754,10 +22754,11 @@ xref: SCTID:198112004 {source="DOID:12698"}
 xref: SCTID:266646002 {source="DOID:12698"}
 xref: SCTID:4754008 {source="MONDO:equivalentTo", source="DOID:12698"}
 xref: UMLS:C0018418 {source="NCIT:C3073", source="MONDO:equivalentTo", source="DOID:12698"}
-is_a: MONDO:0002145 {source="DOID:12698"} ! difference of sexual differentiation
+is_a: MONDO:0002145 {source="DOID:12698"} ! disorder of sexual differentiation
 is_a: MONDO:0002657 {source="MESH:D006177", source="NCIT:C3073/inferred"} ! breast disorder
 intersection_of: MONDO:0001100 ! hypertrophy of breast
 intersection_of: disease_has_location UBERON:0003101 ! male organism
+property_value: IAO:0000233 https://github.com/monarch-initiative/mondo/issues/4949
```

Comment:
we have to add type xsd:anyURI with the term tracker items too (this is a new thing)

<img width="238" alt="image" src="https://user-images.githubusercontent.com/6722114/190232776-9400a99d-d9f1-46d2-9b69-d2c25c711bc9.png">

Reply:
what!? oh, I guess it is because it is not included in the AP "conditions"...

Reply:
What is the difference between using the IRI Editor (where you add the IRI) and using "literal" (where you have to pick the xsd:anyURI)? 
I am assuming that in this case I have to use the "literal"?